src/browser/components/TabBar.jsx

```diff
@@ -71,13 +74,15 @@ export default class TabBar extends React.Component { // need "this"
           title='Add new server'
           draggable={false}
         >
-          <Glyphicon glyph='plus'/>
+          <div className='TabBar-tabSeperator'>
+            {'+'}
```

Comment:
are we no longer using fonts for icons?

Reply:
Not across the board, but the new design uses just a generic `+`